addons/languages/bribri/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
The first publish is manual. Set this to `false` and add a reason (see https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/ae1a2ba7b05a200daf384127a5660f4b22c373a3/addons/languages/songhay/apk/build.gradle#L2)

While you at it, want to update the guides for adding language packs?

Reply:
done.